sound/soc/intel/boards/sof_sdw_rt1308.c

```diff
@@ -11,9 +11,215 @@
 #include <sound/soc.h>
 #include <sound/soc-acpi.h>
 #include <sound/soc-dapm.h>
+#include <linux/soundwire/sdw.h>
+#include <linux/soundwire/sdw_type.h>
+#include <linux/dmi.h>
 #include "sof_sdw_common.h"
 #include "../../codecs/rt1308.h"
 
+#define RT1308_MAX_BQ_REG 480
+
+static const unsigned char dell_0a5d_bq_params[] = {
```

Comment:
nit-pick: I would have used u8 for consistency with the device property definition but that works.

Reply:
also maybe move this table to a new file which will only contain coefficients. The C file should be more about the programming sequences, the binary data should be in a separate file IMHO.

Reply:
Sure,  will move it to a separate .h file.